src/Raven.Server/Json/BlittableJsonTextWriterExtensions.cs

```diff
@@ -1781,7 +1781,14 @@ private static void WriteDocumentProperties(this AbstractBlittableJsonTextWriter
 
             if (first == false)
                 writer.WriteComma();
+
             WriteMetadata(writer, document, metadata, filterMetadataProperty);
+
+            if (document._timeSeriesStream != null)
+            {
+                writer.WriteComma();
+                writer.WriteArray(document._timeSeriesStream.Key, document._timeSeriesStream.TimeSeries, context);
```

Comment:
What is the 'Key' ?

Reply:
It is the magic `Constants.TimeSeries.QueryFunction` prefix, which we return to identify this as TS result.

Reply:
Why we need this? You know what you are streaming in the Client API